src/Settings/Mods.ts

```diff
@@ -309,9 +309,13 @@ export function GenerateModsSettings(): GameStartr.IModAttachrCustoms {
                         for (const actor of opponent.actors) {
                             actor.level += playerPokemonAvg - enemyPokemonAvg;
 
-                            for (const statistic of statistics) {
-                                (actor as any)[statistic] = (actor as any)[statistic + "Normal"] =
-                                    this.MathDecider.compute("pokemonStatistic", actor, statistic);
+                            if (this.utilities.shouldEvolve(actor)) {
+		                        this.utilities.evolvePokemon(actor);
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6758185/19629756/6d9b29ee-9949-11e6-9230-94c81f72acbc.png)

For some reason the indentation is weird only on GitHub.